packages/client.test.ontology/turbo.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["//"],
```

Comment:
Improving build perf by separating the codegen dependencies to per project. You'll see a handful of turbo.json files in here because of that.